src/lib/js/editor.js

```diff
@@ -41,7 +41,11 @@ export class FormeoEditor {
     this.tooltip = new SmartTooltip()
 
     // Load remote resources such as css and svg sprite
-    document.addEventListener('DOMContentLoaded', this.loadResources.bind(this))
+    if (document.readyState === 'loading') {
+      document.addEventListener('DOMContentLoaded', this.loadResources.bind(this))
+    } else {
+      this.loadResources()
+    }
```

Comment:
this is the fix, the cleanFormData changes are to ensure formData intake is the same for renderer and editor